src/components/Team/teamDataColumns.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import heroes from 'dotaconstants/build/heroes.json';
+import strings from 'lang';
+import { transformations, subTextStyle, getTeamLogoUrl } from 'utility';
+import { TableLink } from 'components/Table';
+import constants from '../constants';
+import { TableRow, TableImage } from './TeamStyled';
+import proPlayerImages from './proPlayerImages';
+
+const displayResult = (row) => {
+  const won = row.radiant_win === row.radiant;
+  const string = won ? strings.td_win : strings.td_loss;
+  const textColor = won ? constants.colorGreen : constants.colorRed;
+  return (
+    <span style={{ color: textColor }}>{string}</span>
+  );
+};
+
+const getPlayerImageUrl = (accountId) => {
+  if (proPlayerImages.indexOf(accountId) !== -1) {
```

Comment:
for large N using an object is faster than indexOf on an array, but with a static N here it may not be an issue.